### DISCUSSION TITLE
use edge atrribute in HGT

### DISCUSSION THREAD

**Discussion by Everyday-seu** [pyg-team/pytorch_geometric#4364] [Q&A]
Thanks for your fantastic jobs about pytorch_geometric!!!
Notice that existing HGTConv method uses x_dict,edge_index_dict, but I want to use edge attribute as well.
What should I do?
Look forward to your reply!

**Comment by rusty1s** [comment]
Currently, `HGTConv` does not support edge features. We would need to add support for this. As an alternative, you can take a lot of `HeteroConv` which lets you pick any PyG GNN operator (even the ones that can leverage edge features) and makes it usable within a heterogeneous graph scenario.
